shipyard/README.md

```diff
@@ -0,0 +1,62 @@
+# Shipyard
+
+## Overview
+
+Shipyard enables teams by providing on-demand (ephemeral) environments for every code change (PR). We lower teams' preproduction cloud costs by 70%, increase developer velocity by 50%, and give DevOps/SRE ~30% of their week back! Shipyard works in your existing CI workflow, empowering teams to test before they merge.
```

Comment:
```suggestion
Shipyard enables teams by providing on-demand (ephemeral) environments for every code change (PR). Shipyard works in your existing CI workflow, empowering teams to test before they merge.
```
We generally avoid marketing-style copy in this repository, so removing this sentence